src/xcode/ENA/ENAUITests/AntigenTestProfile/ENAUITests_12_AntigenTestProfile.swift

```diff
@@ -133,9 +133,6 @@ class ENAUITests_12_AntigenTestProfile: CWATestCase {
 		XCTAssertTrue(app.buttons[AccessibilityIdentifiers.ExposureSubmission.AntigenTest.Profile.editButton].waitForExistence(timeout: .long))
 
 		app.swipeUp(velocity: .slow)
```

Comment:
```suggestion
```
I guess it doesn't make sense to swipe up anymore to then just tap the close button

Reply:
@nickguendling  you are right, done